### ISSUE TITLE
Docs?

### ISSUE THREAD

**Issue by spacebiscuit** [t73biz/cakephp2-jwt-auth#6]
Hi,

Is there any documentation for your plugin? I don't have access to configure my server with the dependencies requires to run the Tests so I can't see any example of how to use this.

Thanks in Advance,

Bob.

**Comment by t73biz** [comment]
Hey there.

I don't currently have any docs, but give a couple of days and I can get them together.

Thanks for the interest.

**Comment by spacebiscuit** [comment]
I managed to get it working, but since I didn't install Cake with composer I was unable to install this plugin with composer neither. I had to manually app::import the Vendor file:

App::import('Vendor', 'firebase/php-jwt/src/JWT');

Then I had to comment out the namespace line in JWT.php:

namespace Firebase\JWT;

This enabled me to call the code and decode functions which all works nicely.

Thanks!

**Comment by VaelVictus** [comment]
I had no trouble installing through composer, just put 
`{ "require": { "t73biz/cakephp2-jwt-auth": "1.0.6" } }` into composer.json under /app

However, I'd really like an example of how to use this plugin. Surely someone has some code lying around that you could upload after taking out the sensitive bits? Even if you post basic usage right in this comment section.

**Comment by VaelVictus** [comment]
**I'd recommend against using this library for now,** clearly whoever's in charge is not interested. There's a valuable pull request that has some documentation if you're bent on using this library. The request is being completely ignored by the owner of this project. :}

Also, it does not install the latest update to firebase's JWT.php when composing. This version includes an exploit involving empty keys.

**To get it working for myself, I did the following:**
- installed JWT.php with: composer require firebase/php-jwt
- commented out the namespace as spacebiscuit said (THANK YOU!)
- opened JWT.php and in decode, I turned all exceptions into die() because Cake was getting fussy

And I followed this tutorial.
https://www.sitepoint.com/php-authorization-jwt-json-web-tokens/

I cannot say with confidence that I've done the best job, but JWTs are working for me without having to use a Cake plugin. My requirements are basic and may not mirror your company's needs.
